system-test/compute.js

```diff
@@ -693,70 +694,44 @@ describe('Compute', () => {
 
   describe('rules', () => {
     const RULE_NAME = generateName('rule');
-    const rule = compute.rule(RULE_NAME);
-    const service = compute.service(generateName('service'));
-    const INSTANCE_GROUP_NAME = generateName('instance-group');
-    const HEALTH_CHECK_NAME = generateName('health-check');
-
-    // To create a rule, we need to also create a TargetHttpProxy and UrlMap.
-    // Until they are officially supported by google-cloud-node, we make manual
-    // requests to create and delete them.
-    const TARGET_PROXY_NAME = generateName('target-proxy');
-    const URL_MAP_NAME = generateName('url-map');
+    const NETWORK_NAME = generateName('network-name');
+    const SUBNETWORK_NAME = generateName('subnetwork-name');
+    const BACKEND_SERVICE_NAME = generateName('backend-service-name');
 
     before(async () => {
-      await createService(service.name, INSTANCE_GROUP_NAME, HEALTH_CHECK_NAME);
-      await createUrlMap({
-        name: URL_MAP_NAME,
-        defaultService: 'global/backendServices/' + service.name,
-      });
-      await createTargetProxy({
-        name: TARGET_PROXY_NAME,
-        urlMap: 'global/urlMaps/' + URL_MAP_NAME,
-      });
-      await create(rule, {
-        target: 'global/targetHttpProxies/' + TARGET_PROXY_NAME,
-        portRange: '8080',
-        IPProtocol: 'TCP',
-      });
+      execSync(
+        `gcloud compute networks create ${NETWORK_NAME} --subnet-mode=custom`
+      );
+      execSync(
+        `gcloud compute networks subnets create ${SUBNETWORK_NAME} --network=${NETWORK_NAME} --range=10.0.1.0/24 --region=us-central1`
+      );
+      execSync(
+        `gcloud compute backend-services create ${BACKEND_SERVICE_NAME} --load-balancing-scheme=INTERNAL --region=us-central1`
+      );
+      execSync(
+        `gcloud compute forwarding-rules create ${RULE_NAME} --load-balancing-scheme=INTERNAL --backend-service=${BACKEND_SERVICE_NAME} --subnet=${SUBNETWORK_NAME}  --network=${NETWORK_NAME} --region=us-central1 --ports=80-82`
+      );
     });
 
-    it('should get a list of rules', async () => {
-      const [rules] = await compute.getRules();
-      assert(rules.length > 0);
+    after(async () => {
+      const [firewalls] = await compute.getFirewalls();
+      const firewallsToDelete = firewalls
+        .filter(x => x.name.includes('network-name'))
+        .map(y => y.name);
+      for (const firewall of firewallsToDelete) {
+        await deleteFirewallRule(firewall);
```

Comment:
Would the method `firewall.delete()` work? (You'd have to remove the `.map(y => y.name)` part above)